server/controllers/visitation.controllers.js

```diff
@@ -1,15 +1,15 @@
-import scheduleVisit from '../services/visit.service';
+import scheduleVisitation from '../services/visitation.service';
 // import { sendMail } from '../services/mailer.service';
 // import EMAIL_CONTENT from '../../mailer';
 import httpStatus from '../helpers/httpStatus';
 
-const VisitController = {
+const VisitationController = {
   book(req, res, next) {
     const booking = req.locals;
     const { user } = req;
-    scheduleVisit({ ...booking, visitorId: user._id })
+    scheduleVisitation({ ...booking, userId: user._id })
       .then((schedule) => {
-        // sendMail(EMAIL_CONTENT.ACTIVATE_YOUR_ACCOUNT, user, {
+        // sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, user, {
```

Comment:
For me, I would create a separate variable to build the contentBottom for the schedule mail, something like

```js
const contentBottom  = `
<strong> Name: </strong> ${schedule.visitorName}
// other variables here
...

```
Note: BALLERS_EMAIL would have been defined in a separate file it i.e constants.js

```suggestion
        // const contentBottom = ... 
        sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, { email: BALLERS_EMAIL }, { contentBottom })
```

This would be sent to mailtrap.io when you are testing on your local machine and would send to the email address in OUR_EMAIL when we have uploaded it.